wordpress-mobile/WordPress-iOS
# [Site creation] First draft of the Site Info screen

Implements #10324 

In order to continue moving forward, and have at UI for all the steps of the flow (although still in a very early draft stage) here is the Site Info.

There is no logic to activate / deactivate the Skip / Next button yet, or to validate the data.

To test:
- The tests should pass
- Navigate to My Sites > Tap + > Enhanced site creation
- The Site Info step should be part of the flow. 

Next steps: remove boilerplate and start polishing all the UI, integrating the proper cells, applying styles...
